### PR TITLE
docs: portuguese translation

### PR DESCRIPTION
**Pull request by Sn0wye**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog
- [x]  CT3A
    - [x]  Intro
    - [x]  Why
    - [x]  Install
    - [x]  Folder Structure
    - [x]  FAQ
    - [x]  T3 Collection
    - [x]  Other Recommendations
- [x]  Usage
    - [x]  First Steps
    - [x]  Next
    - [x]  Typescript
    - [x]  tRPC
    - [x]  Prisma
    - [x]  NextAuth.js
    - [x]  Env
    - [x]  Tailwind
- [x]  Deploy
    - [x]  Vercel
    - [x]  Docker
---

## Rephrase && Review
- [x] Config
- [x]  CT3A
    - [x]  Intro
    - [x]  Why
    - [x]  Install
    - [x]  Folder Structure
    - [x]  FAQ
    - [x]  T3 Collection
    - [x]  Other Recommendations
- [x]  Usage
    - [x]  First Steps
    - [x]  Next
    - [x]  Typescript
    - [x]  tRPC
    - [x]  Prisma
    - [x]  NextAuth.js
    - [x]  Env
    - [x]  Tailwind
- [x]  Deploy
    - [x]  Vercel
    - [x]  Docker

### CODE REVIEW

**Comment by juliusmarminge** [comment]
Please check out the new [TRANSLATIONS.md](https://github.com/t3-oss/create-t3-app/blob/next/www/TRANSLATIONS.md) and the notes in [config.ts](https://github.com/t3-oss/create-t3-app/blob/next/www/src/config.ts) and configure the sidebar for your language.

**Comment by Sn0wye** [comment]
> 

I Forgot to do that! fixed with commit [c5edf37](https://github.com/t3-oss/create-t3-app/pull/853/commits/c5edf37b9b27294e4d04d3e2caa0e09346bf7943)

**Comment by nexxeln** [comment]
Awesome! Ready for review?

File: www/src/pages/pt/faq.md
Comment (victoriaquasar) [inline www/src/pages/pt/faq.md:24]: Ambiguous phrase? Maybe `recomendam que você aprenda a stack como um todo, enquanto desenvolve a utilizando.`

File: www/src/pages/pt/folder-structure.md
Comment (victoriaquasar) [inline www/src/pages/pt/folder-structure.md:10]:
```suggestion
A seguir, temos a estrutura de pastas de um novo T3 app inicializado, com todas as opções selecionadas. A descrição de cada pasta indica seu propósito e se é apenas incluída com bibliotecas selecionadas.
```

File: www/src/pages/pt/installation.md
Comment (victoriaquasar) [inline www/src/pages/pt/installation.md:20]: Remove `yarn` option from the documentation. See #454 (wontfix).

File: www/src/pages/pt/introduction.md
Comment (victoriaquasar) [inline www/src/pages/pt/introduction.md:40]:
```suggestion
## Tipos seguros e bem definidos não são opcionais
```

File: www/src/pages/pt/usage/tailwind.md
Comment (victoriaquasar) [inline www/src/pages/pt/usage/tailwind.md:31]: This phrase is in english

File: www/src/pages/pt/usage/tailwind.md
Comment (victoriaquasar) [inline www/src/pages/pt/usage/tailwind.md:37]: This phrase is in english

File: www/src/pages/pt/why.md
Comment (victoriaquasar) [inline www/src/pages/pt/why.md:20]: Text is in english

```suggestion
`Typesafety` torna as coisas mais rápidas. Se isso não te convenceu, você [pode estar usando TypeScript errado...](https://www.youtube.com/watch?v=RmGHnYUqQ4k)
```

File: www/src/pages/pt/why.md
Comment (victoriaquasar) [inline www/src/pages/pt/why.md:18]: Remove this wrongly placed phrase.

File: www/src/pages/pt/usage/typescript.md
Comment (victoriaquasar) [inline www/src/pages/pt/usage/typescript.md:39]:
```suggestion
É, talvez, a ferramenta que mais oferece produtividade aos desenvolvedores; fornecer documentação do código que você está escrevendo ou consumindo diretamente em seu editor e ter feedback instantâneo quando você inevitavelmente cometer erros é absolutamente inestimável.
```

File: www/src/pages/pt/usage/tailwind.md
Comment (Sn0wye) [reply www/src/pages/pt/usage/tailwind.md]: solved in commit #853

File: www/src/pages/pt/usage/tailwind.md
Comment (Sn0wye) [reply www/src/pages/pt/usage/tailwind.md]: solved in commit [a308d54](https://github.com/t3-oss/create-t3-app/pull/853/commits/a308d54a5e8f7957194b870602fa8cf2728f7e7a)

**Comment by Sn0wye** [comment]
> Awesome! Ready for review?

Ready! Brazilian and Portuguese guys, feel free to request changes and review (I may have committed some errors), all improvements are welcome!

**Comment by Sn0wye** [comment]
Hey @juliusmarminge can you check the ci build problem in www/src/components/navigation/leftSidebar.astro file?

**Comment by nexxeln** [comment]
> Hey @juliusmarminge @nexxeln @c-ehrlich can you check the ci build problem in www/src/components/navigation/leftSidebar.astro file?

It's being fixed at the moment! @juliusmarminge is working hard on it. CI stuff is super hard to debug haha

**Comment by minsk-dev** [comment]
> > Awesome! Ready for review?
> 
> Ready! Brazilian and Portuguese guys, feel free to request changes and review (I may have committed some errors), all improvements are welcome!

I'm reviewing everything. Takes a lot of time so I don't know when I'll submit

**Comment by Sn0wye** [comment]
> > > Awesome! Ready for review?
> > 
> > Ready! Brazilian and Portuguese guys, feel free to request changes and review (I may have committed some errors), all improvements are welcome!
> 
> I'm reviewing everything. Takes a lot of time so I don't know when I'll submit 

Take your time!

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:17]:
```suggestion
### 1. Configuração do Next
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:12]:
```suggestion
Por favor, note que o Next.js requer um processo diferente para tempo de build (disponível no frontend, prefixado por `NEXT_PUBLIC`) e ambiente de tempo de execução, somente do lado do servidor, variáveis. Nesta demonstração estamos usando duas variáveis. Preste atenção em suas posições no `Dockerfile`, argumentos de linha de comando e `docker-compose.yml`:
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:19]:
```suggestion
Em seu arquivo [`next.config.mjs`](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs), configure a opção `output` como `standalone` para [reduzir o tamanho de imagens aproveitando automaticamente os rastreamentos de saída](https://nextjs.org/docs/advanced-features/output-file-tracing):
```

I am not completely satisfied with the expression "aproveitando automaticamente os rastreamentos de saída". 
I do not think the current expression is clear enough in Portuguese to understand the meaning.
If someone has a better expression, please suggest.

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:33]:
```suggestion
      Clique aqui e inclua esse conteúdo no arquivo <code>.dockerignore</code>:
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:29]:
```suggestion
### 2. Criação do arquivo .dockerignore
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:52]:
```suggestion
### 3. Criação do arquivo Dockerfile
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:54]:
```suggestion
> Como não estamos puxando as variáveis de ambiente do servidor para o nosso contêiner, a [validação do esquema do ambiente](/en/usage/env-variables) falhará. Para evitar isso, temos que adicionar a flag `SKIP_ENV_VALIDATION=1` ao comando de build para que os esquemas env não sejam validados no momento de build.
```

I changed in the suggestion `compilação` to `build`, because I thought in other places even in Portuguese in this PR, the word "build" is kept as is.
But I am not sure which would be better.

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:58]:
```suggestion
      Clique aqui e inclua esse conteúdo no arquivo <code>Dockerfile</code>:
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:129]:
```suggestion
> **_Observações_**
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:133]:
```suggestion
> - _Next.js coleta [dados anônimos de telemetria sobre uso geral](https://nextjs.org/telemetry). Remova o primeiro comentário de `ENV NEXT_TELEMETRY_DISABLED 1` para desabilitar a telemetria durante o build. Remova o segundo comentário para desabilitar a telemetria durante o tempo de execução._
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:138]:
```suggestion
## Execute build e run da imagem localmente
```
I opted for suggesting to keep the words "build" and "run" because they are the same as the commands.

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:140]:
```suggestion
Execute `build` e `run` desta imagem localmente com os seguintes comandos:
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:155]:
```suggestion
      Siga as etapas 1 a 4 acima, clique aqui e inclua o conteúdo no arquivo <code>docker-compose.yml</code>:
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:190]:
```suggestion
Você pode usar [deploys automáticos de Dockerfile](https://docs.railway.app/deploy/dockerfiles) em uma PaaS como a [Railway](https://railway.app) para fazer deploy das suas aplicações. Se você tiver o [CLI da Railway instalado](https://docs.railway.app/develop/cli#install), poderá fazer deploy da sua aplicação com os seguintes comandos:
```

File: www/src/pages/pt/deployment/docker.md
Comment (claudiofreitas) [inline www/src/pages/pt/deployment/docker.md:200]:
```suggestion
Vá para "Variables" e inclua seu `DATABASE_URL`. Em seguida, vá para "Settings" e selecione "Generate Domain". Para ver um exemplo em execução no Railway, visite [ct3a-docker.up.railway.app](https://ct3a-docker.up.railway.app/).
```

I changed back the words to English because the user interface of Railway is in English.

**Review by claudiofreitas — COMMENTED** [review]
Hi!
I reviewed the files
```
www/src/config.ts
www/src/pages/pt/deployment/docker.md
```
and included some suggestions.
Some of the suggestions I included because I thought they would read more naturally in Portuguese.
And some points I was not completely sure and included some comment together with the suggestions.

I could not review all the files, maybe someone else can help with the remaining ones.

**Review by minsk-dev — COMMENTED** [review]
Submiting this but it's not a full review yet

File: www/src/pages/pt/deployment/docker.md
Comment (Sn0wye) [reply www/src/pages/pt/deployment/docker.md:19]: Liked that one, gonna commit

File: www/src/pages/pt/deployment/docker.md
Comment (Sn0wye) [reply www/src/pages/pt/deployment/docker.md:138]: Fair enough

File: www/src/pages/pt/faq.md
Comment (Sn0wye) [reply www/src/pages/pt/faq.md:28]: Todes daí forçou, desculpa aí, todos inlcui o feminino também

Comment (Sn0wye) [reply www/src/pages/pt/faq.md]: Resolvido no commit #e87523b

**Comment by c-ehrlich** [comment]
don't worry about typecheck failing after i merged next, this is a known issue. will hopefully have a fix asap :)

File: www/src/pages/pt/deployment/docker.md
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/docker.md:69]:
```suggestion
# Instalação do Prisma Client - remova se não estiver usando o Prisma
```

File: www/src/pages/pt/deployment/docker.md
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/docker.md:73]:
```suggestion
# Instalação de dependências com base no package manager padrão
```

File: www/src/pages/pt/deployment/vercel.md
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/vercel.md:23]:
```suggestion
## Usando o painel de controle da Vercel
```

File: www/src/pages/pt/deployment/vercel.md
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/vercel.md:59]:
```suggestion
Após o primeiro deploy, esse comando o fará em um branch de preview. Você precisará incluir `--prod` para enviar alterações diretamente para a produção.
```

**Comment by dBianchii** [comment]
Hi, Olá! Boa noite, good night. I would like to contribute to this. Is it just reviews that are needed?

**Comment by juliusmarminge** [comment]
Conflicts

**Comment by Sn0wye** [comment]
> Hi, Olá! Boa noite, good night. I would like to contribute to this. Is it just reviews that are needed?

For now, yes!

**Comment by Sn0wye** [comment]
I've reviewed that by myself, as I've seen, everything looks great.
Just waiting for one or two reviewers than proceed with the merge.

**Comment by Sn0wye** [comment]
> Conflicts

No more my man, no more

File: www/src/pages/pt/faq.md
Comment (victoriaquasar) [inline www/src/pages/pt/faq.md:57]:
```suggestion
Conforme o [Axioma T3 #2](/pt/introduction#agir-com-responsabilidade), nós amamos _tecnologia de ponta_, porém valorizamos a estabilidade, nosso roteador inteiro é díficil de mover, [não é um bom lugar para utilizar _bleeding edges_](https://youtu.be/mnwUbtieOuI?t=1662). Enquanto `/app` é [instável e experimental](https://youtu.be/rnsC-12PVlM?t=818), não está pronto para produção; a API está em beta e é esperado que possua *breaking changes*.
```

File: www/src/pages/pt/deployment/docker.md
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/docker.md:19]:
```suggestion
Em seu arquivo [`next.config.mjs`](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs), configure a opção `output` como `standalone` para [reduzir o tamanho de imagens](https://nextjs.org/docs/advanced-features/output-file-tracing):
```

Over-information is not needed in translations, the target link already explains the process.

**Review by victoriaquasar — APPROVED** [review]

**Comment by Sn0wye** [comment]
Hey @juliusmarminge @nexxeln, ready to merge!

**Review by c-ehrlich — APPROVED** [review]
🎉

**Pull request merged**
